typings/index.d.ts

```diff
@@ -49,10 +48,10 @@ import {
   RESTPostAPIApplicationCommandsJSONBody,
   Snowflake,
 } from 'discord-api-types/v9';
+import { Response } from 'node-fetch';
 import { ChildProcess } from 'node:child_process';
 import { EventEmitter } from 'node:events';
 import { AgentOptions } from 'node:https';
-import { Response } from 'node-fetch';
```

Comment:
Sure…?

Reply:
It was done by ts organize import which organized it alphabetically. You can check it's preset in line 51. Idk which convention do you use. Should I turn that off?